### PR TITLE
Contract test for additional identifiers

### PR DESCRIPTION
**Pull request by prerna-p**
*Issue #, if available:* 

*Description of changes:*
Adding a validation check for additional identifiers for a resource

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/rpdk/core/contract/resource_client.py [hunk 1]
```diff
@@ -188,6 +188,7 @@ def _update_schema(self, schema):
             {fragment_decode(prop, prefix="") for prop in identifier}
             for identifier in additional_identifiers
         ]
+        self.additional_identifiers_paths = self._additional_identifiers_paths
```
Comment (anshikg) [inline src/rpdk/core/contract/resource_client.py:191]: you can just rename _additional_identifiers_paths to additional_identifiers_paths

File: src/rpdk/core/contract/suite/handler_commons.py [hunk 2]
```diff
@@ -176,3 +176,15 @@ def test_input_equals_output(resource_client, input_model, output_model):
     )
 
     assert pruned_input_model == pruned_output_model
+
+
+# TODO: add decorator for checking that resource model has additional identifiers
+def test_read_success_additional_identifier(resource_client, current_resource_model):
+    additional_identifier_only_model = create_model_with_properties_in_path(
+        current_resource_model.copy(),
```
Comment (anshikg) [inline src/rpdk/core/contract/suite/handler_commons.py:184]: Copy is not required. This method does not modify the model

File: src/rpdk/core/contract/suite/handler_create.py [hunk 3]
```diff
@@ -19,21 +19,28 @@
...
+            _status,
+            response,
+            _error,
+        ) = resource_client.call_and_assert(
             Action.CREATE, OperationStatus.SUCCESS, request
         )
         model = response["resourceModel"]
         test_input_equals_output(resource_client, input_model, model)
-        yield model, request
+        yield model, request  #
```
Comment (anshikg) [inline src/rpdk/core/contract/suite/handler_create.py:43]: please remove "#"

**Review by anshikg — COMMENTED** [review]
Please add a little more description to the details section

**Comment by prerna-p** [comment]
Will be closing this PR as I need to fork the repo. Will incorporate the review comments in new PR.

**Pull request closed without merging**